src/buttons/label.ts

```diff
@@ -0,0 +1,30 @@
+import {Directive, Renderer2, ElementRef} from '@angular/core';
+
+@Directive({selector: '[ngbButtonLabel]', host: {'[class.btn]': 'true'}})
+export class NgbButtonLabel {
+  constructor(private _renderer: Renderer2, private _elRef: ElementRef) {}
```

Comment:
True that we could have done it with 3 fields and host bindings, don't really need Renderer, right?